src/applications/fry-dea/components/ConfirmationResponses.jsx

```diff
@@ -28,13 +30,13 @@ const ConfirmationReceiptBox = ({ name }) => {
         <dt>
           <strong>Date received</strong>
         </dt>
-        <dd>{format(new Date(), 'MMM d, yyyy')}</dd>
+        <dd>{format(new Date(), 'MMMM d, yyyy')}</dd>
       </dl>
 
       <button
         type="button"
         onClick={() => window.print()}
-        className="usa-button vads-u-width--auto"
+        className="usa-button vads-u-width--auto vads-u-margin-bottom--0"
```

Comment:
The button's width is auto by default, so `vads-u-width--auto` isn't needed here.